pkgs/tools/text/crowdin-cli/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, gawk
+, git
+, gnugrep
+, jre
+, unzip
+}:
+
+let
+  pname = "crowdin-cli";
+  version = "3.6.4";
+in stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/crowdin/${pname}/releases/download/${version}/${pname}.zip";
+    sha256 = "123mv0s1jppidmwsvr8a6f8429xmpskxmnv4p8jpnfa9zrw86aaw";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  propagatedBuildInputs = [
+      gawk
+    gnugrep
+  git
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D crowdin-cli.jar $out/lib/crowdin-cli.jar
+    install -D crowdin_completion $out/share/bash-completion/completions/crowdin
```

Comment:
Do we have zsh or fish?

Please use installShellCompletion.

Reply:
> Do we have zsh or fish?

only bash is provided

> Please use installShellCompletion.

I love these helpers, thanks!